src/app/clients/view-client/view-client.component.html

```diff
@@ -118,6 +118,7 @@ <h3>{{clientInfo?.displayName}}</h3>
 
               </tr>
             </table>
+            <mat-paginator #loanPaginator [pageSizeOptions]="[10,25,50,100]" showFirstLastButtons></mat-paginator>
```

Comment:
I asked you to ensure that there is a single space between `,` and page size option (Example: 25). Don't remove the space. Update this for all instances.
```suggestion
            <mat-paginator #loanPaginator [pageSizeOptions]="[10, 25, 50, 100]" showFirstLastButtons></mat-paginator>
```